config/application.rb

```diff
@@ -58,13 +58,41 @@ class Application < Rails::Application
     # we do here instead of using initializers because of a Rails 5.1 vs
     # MySQL bug where `rake db:reset` causes ActiveRecord to be loaded
     # before initializers and causes configuration not to be respected.
-    config.load_defaults 6.1
+    config.load_defaults 7.0
+
+    # TODO: consider removing this to enable the default value 'true', and setting `allow_other_host: true` for `redirect_to` only where needed
+    # Protect from open redirect attacks in `redirect_back_or_to` and `redirect_to`.
+    config.action_controller.raise_on_open_redirects = false
+
+    # Enable parameter wrapping for JSON.
+    # Previously this was set in an initializer. It's fine to keep using that initializer if you've customized it.
+    # To disable parameter wrapping entirely, set this config to `false`.
+    config.action_controller.wrap_parameters_by_default = false
```

Comment:
So, this is related to the removal of `config/initializers/wrap_parameters.rb`.

It seems that we were disabling parameter wrapping in that initializer explicitly, and now it's enough to just set this setting to `false`.

I am not sure why we do this, but maybe this is because in each controller this is handled differently - we add `wrap_parameters` explicitly.

More about parameters wrapping: https://api.rubyonrails.org/v8.0.1/classes/ActionController/ParamsWrapper.html

And the config details: https://guides.rubyonrails.org/configuring.html#config-action-controller-wrap-parameters-by-default

Reply:
Actually, now it seems that it's not needed... I thought I had tried previously and it failed in some tests :thinking: Maybe false memories.